src/App.js

```diff
@@ -1,19 +1,30 @@
-import './App.css';
-import Nav from './components/Nav';
+import React, { useContext } from 'react';
 import Login from './components/Login';
+import { BrowserRouter as Router, Routes, Route, Link } from 'react-router-dom';
 import Register from './components/Register';
-import { Routes, Route } from 'react-router-dom';
-
+import Nav from './components/Nav';
+import Profile from './components/Profile/Profile';
+import { ChakraProvider } from '@chakra-ui/react';
+import { Box } from '@chakra-ui/react';
+import { AuthContext } from './components/UserContext';
 function App() {
+  const { providerValue } = useContext(AuthContext);
+  console.log('test', providerValue);
   return (
-    <div className="App">
-      <Nav />
-      <Routes>
-        <Route path="/" element={<Login />} />
-        <Route path="register" element={<Register />} />
-      </Routes>
-    </div>
+    <ChakraProvider>
+      <Box backgroundColor="gray.200" idth="100wh" minH="100vh">
+        <Router>
+          <Nav></Nav>
```

Comment:
Components that don't have children should be self closing
```suggestion
          <Nav/>
```